Resources/Prototypes/_RMC14/Entities/Objects/Weapons/Guns/Revolvers/mateba.yml

```
@@ -15,10 +15,16 @@
     soundGunshot:
       path: /Audio/_RMC14/Weapons/Guns/Gunshots/gun_mateba.ogg
   - type: RevolverAmmoProvider
+    whitelist:
+      tags:
+      - RMCSpeedLoaderMateba
+      - RMCCartridgeRevolverMateba
     proto: RMCCartridgeRevolverMateba
     capacity: 6
     chambers: [ True, True, True, True, True, True ]
     ammoSlots: [ null, null, null, null, null, null ]
+  - type: GunDamageModifier
+    multiplier: 1.2 # TODO: set this to 1.5 once GunDamageModifier system is fixed
```

This is fixed now with https://github.com/RMC-14/RMC-14/commit/e331f3e327e8bd588c1c6b7bacdee666d57dd750
```suggestion
    multiplier: 1.5
```